### DISCUSSION TITLE
Operator do-while vs while

### DISCUSSION THREAD

**Discussion by willardwang** [Netflix/conductor#3747] [Ideas]
hi，Whether it is possible to support  while directly, instead of using do-while+switch to implement while in disguise。From programming experience, while is more commonly used than do-while。

**Comment by willardwang** [comment]
For example, if I want to loop through an array, in order to avoid an exception when the array length is 0, I need to determine the length with the switch and then use the do-while

**Comment by v1r3n** [comment]
Hi @willardwang I think this will be a good feature addition. We will prioritize this and if you have ideas happy to collaborate on this.
